persist/sqlite/v2consensus.go

```diff
@@ -264,28 +264,44 @@ func updateV2FileContractElements(tx *txn, revert bool, index types.ChainIndex,
 }
 
 func updateV2FileContractIndices(tx *txn, revert bool, index types.ChainIndex, fces []explorer.V2FileContractUpdate) error {
-	resolutionIndexStmt, err := tx.Prepare(`UPDATE v2_last_contract_revision SET resolution_type = ?, resolution_height = ?, resolution_block_id = ?, resolution_transaction_id = ? WHERE contract_id = ?`)
+	resolutionIndexStmt, err := tx.Prepare(`UPDATE v2_last_contract_revision SET resolution_type = ?, resolution_height = ?, resolution_block_id = ?, resolution_transaction_id = ?, renewed_to_id = ? WHERE contract_id = ?`)
 	if err != nil {
 		return fmt.Errorf("updateV2FileContractIndices: failed to prepare resolution index statement: %w", err)
 	}
 	defer resolutionIndexStmt.Close()
 
+	renewedFromStmt, err := tx.Prepare(`UPDATE v2_last_contract_revision SET renewed_from_id = ? WHERE contract_id = ?`)
+	if err != nil {
+		return fmt.Errorf("updateV2FileContractIndices: failed to prepare renewed from statement: %w", err)
+	}
+	defer renewedFromStmt.Close()
+
 	for _, update := range fces {
 		// id stays the same even if revert happens so we don't need to check that here
 		fcID := update.FileContractElement.ID
 
 		if revert {
```

Comment:
I'm assuming it's fine because of the tests you added, but could you explain a bit why the revert branch doesn't look like a mirrored copy from the apply branch? You'd expect a revert to exactly revert the apply right? so how can it be done in one statement while apply needs two statements?